### ISSUE TITLE
Platform test on Azure fails due to a timeout in creating a ssh connection to the VM under test in azure

### ISSUE THREAD

**Issue by Vincinator** [gardenlinux/gardenlinux#1842]
**What happened:**

The azure platform tests run by the nightly github action failed.
The test-VM in azure is not booting fast enough for our timeout.

```
paramiko.transport:transport.py:1893 Auth banner: b'"System is booting up. Unprivileged users are not permitted to log in yet. Please come back later. For technical details, see pam_nologin(8)."\n'
```

-  [Call to wait_systemd_boot](https://github.com/gardenlinux/gardenlinux/blob/90300db8ec95a0f2b7857b0821fd793a16d9532e/tests/integration/azure.py#L451)
- [wait_systemd_boot(client) definition](https://github.com/gardenlinux/gardenlinux/blob/90300db8ec95a0f2b7857b0821fd793a16d9532e/tests/helper/utils.py#L119)

Ultimately, the tests do not start until this synchronously blocking command is done. 
```
systemctl is-system-running --wait
```
see systemctl [docs](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#--wait)

**Debug Logs:**
- 1312.1 build pipeline: https://github.com/gardenlinux/gardenlinux/actions/runs/6955241114/job/18925068987#step:13:99

**Environment:**
- GL 1312.1 in rel-1312 branch

**Comment by Vincinator** [comment]
Azure Issue, won't fix in garden linux pipeline - currently no issues. We could implement further workarounds (retry or increase wait).. remove this issue
